app/src/main/java/net/skyscanner/backpack/demo/ComponentDetailActivity.kt

```diff
@@ -0,0 +1,65 @@
+package net.skyscanner.backpack.demo
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.support.v7.widget.Toolbar
+import android.view.View
+import android.widget.ImageView
+import com.squareup.picasso.Picasso
+import net.skyscanner.backpack.demo.data.ComponentRegistry
+
+/**
+ * An activity representing a single Component detail screen. This
+ * activity is only used on narrow width devices. On tablet-size devices,
+ * item details are presented side-by-side with a list of items
+ * in a [MainActivity].
+ */
+class ComponentDetailActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_component_detail)
+        val toolbar = findViewById<View>(R.id.detail_toolbar) as Toolbar
+        setSupportActionBar(toolbar)
+        val image = findViewById<ImageView>(R.id.img)
+        Picasso.get().load(R.drawable.header).resize(1024, 800)
```

Comment:
I'm not sure about this header anymore. We want to take screenshots of these components to show in the docs site and this header will kinda get in the way.

Reply:
Will remove this header and add toggle for RTL once we add support, screenshot tests and docs.

Reply:
How does an RTL toggle work - can you programmatically trigger that with an Android API? As far as I know on iOS we're just gonna have to set it manually in settings.app

Reply:
This can be done programmatically.

Reply:
Good old Android, not being all locked down!

Reply:
@shaundon join the dark side, we have RTL toggles